src/app/domain/geolocation/hooks/useGeolocationTracking.ts

```diff
@@ -27,7 +27,7 @@ export const useGeolocationTracking = (): void => {
       if (trackingEnabled) {
         await checkGeolocationQuota(client)
       } else {
-        await checkShouldStartTracking()
+        await startTrackingIfAllowed()
```

Comment:
It is not really if "allowed". It is if tracking has been started before, so the shouldStartTracking value is true in local storage but we see when starting the app that the tracking is not started. So tracking is not started even if it should be started.

Reply:
Here "Allowed" is in the meaning that the geolocation was enabled by the user. The `shouldStartTracking` is set to true for the first time when the user enables the toggle, so it is kind of a consent, thus the "allowed". 
So I guess both "allowed" and "shouldStart" are correct here, depending on the perspective.
My concern here was especially on the "check" vs "start", as I do not expect a "check" function to do more than checking something. 
Is `startTrackingIfShouldBe` ok? Sounds a bit weird, but understandable?

Reply:
For me it is not only a consent because is it removed when the user disables tracking.

But totally agree about changing _check_ to _start_. `startTrackingIfShouldBe` is like you said weird but understandable. So clearly better that usual but not understandable.

Reply:
> For me it is not only a consent because is it removed when the user disables tracking.

Well, when you disable the tracking, you kind of remove your consent to be tracked ;)
Anyway, I renamed in `startTrackingIfShouldBe`!